docs_src/microservices/application/services/AppLLRPInventory/ApiReference.md

```diff
@@ -0,0 +1,9 @@
+---
+title: App RFID LLRP Inventory - API Reference
+---
+
+# App RFID LLRP Inventory - API Reference
+
+In addition to the APIs listed below, this service inherits the [common APIs](../../ApiReference.md) from the App SDK.
+
+<swagger-ui src="https://raw.githubusercontent.com/lenny-intel/app-rfid-llrp-inventory/swagger/openapi/v3/app-rfid-llrp-inventory.yaml"></swagger-ui>
```

Comment:
Note this link will change to the main repo once this PR https://github.com/edgexfoundry/app-rfid-llrp-inventory/pull/244 is merged.